.travis.yml

```diff
@@ -67,3 +67,5 @@ jobs:
       env: PLATFORM=icl
```

Comment:
@dianacretu I would prefer to split this into two commits:
- first commit adds imx8x parameter to qemu-check.sh
- second commit adds boot test [ the current commit]